### PR TITLE
About section problem Issue #462

### PR DESCRIPTION
**Pull request by hazemk537**
- Issue #462 resolved
- Two images was not displayed , the  two links was not valid so we changed them

### CODE REVIEW

File: src/components/footer/AboutModal.tsx [hunk 1]
```diff
@@ -32,11 +32,11 @@ const AboutModal = (props: ModalProps): React.ReactElement => {
               <a href="https://developers.google.com/open-source/soc/" rel="noreferrer" target="_blank">
                 <img
                   alt="Google Summer of Code"
-                  src="https://summerofcode.withgoogle.com/static/img/summer-of-code-logo.svg"
+                  src="https://summerofcode.withgoogle.com/assets/media/logo.svg"
                   style={{ height: '2.5em' }}
                 />
               </a>
-              <a href="https://developers.google.com/open-source/gci/" rel="noreferrer" target="_blank">
+              <a href="https://opensource.google/static/images/documentation/reference/growing/GCI-logo.png" rel="noreferrer" target="_blank">
```
Comment (sushain97) [inline src/components/footer/AboutModal.tsx:39]: Why did this link change?

**Review by sushain97 — CHANGES_REQUESTED** [review]

File: src/components/footer/AboutModal.tsx (hunk 1 above)
Comment (hazemk537) [reply src/components/footer/AboutModal.tsx:39]: there is two logos  not appear well 
image

Comment (TinoDidriksen) [reply src/components/footer/AboutModal.tsx:39]: But you changed the link's href, not the image. That's a rather fundamental HTML thing to get wrong.

Also, you should put assets into the repo so we don't depend on 3rd parties.

Comment (hazemk537) [reply src/components/footer/AboutModal.tsx:39]: I am sorry it was a typo, i will update it

**Pull request closed without merging**